coin-flip/index.html

```diff
@@ -1,17 +1,18 @@
 <!DOCTYPE html>
-<html>
+<html lang="en-US">
 <head>
 <title>Coin Flipper | My Digital Spot</title>
 <link rel="stylesheet" href="index.css">
-<script src="script.js"></script>
```

Comment:
I moved `script.js` to the end of your `<body>` tag since that's where you usually will find them. Not a requirement, but can avoid some timing issues.